File: 2678-number-of-senior-citizens/2678-number-of-senior-citizens.js
/**
 * @param {string[]} details
 * @return {number}
 */
var countSeniors = function(details) {
    let count = 0; // Initialize a counter for seniors

    for (let detail of details) {
        // Extract the age from the string (characters at indices 11 and 12)
        let ageStr = detail.substring(11, 13); // The age is represented by the 12th and 13th characters
        let age = parseInt(ageStr); // Convert the age string to an integer
        
        // Check if the age is greater than 60
        if (age > 60) {
            count++; // Increment the counter if the age is greater than 60
        }
    }

    return count; // Return the total count of seniors
};

File: 2678-number-of-senior-citizens/2678-number-of-senior-citizens.py
class Solution:
    def countSeniors(self, details: List[str]) -> int:
        count = 0
        
        for detail in details:
            # Extract the age from the string (characters at indices 11 and 12)
            age_str = detail[11:13]  # The age is represented by the 12th and 13th characters
            age = int(age_str)  # Convert the age string to an integer
            
            # Check if the age is greater than 60
            if age > 60:
                count += 1
                
        return count
